src/websites-2/controller/products.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { productPost, productsGet,ProductGet, productPut, dlProduct } from "../services/products.servives"
```

Comment:
rename the name of the folder don't use the name of the ticket to do this please use something clear